power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/sparse_lu_solver.hpp

```diff
@@ -138,6 +138,7 @@ template <rk2_tensor Matrix> class DenseLUFactor {
                 throw SparseMatrixError{}; // can not specify error code
             }
         }
+        (void)std::move(matrix);
```

Comment:
Why here?

Reply:
this function captures `func` by prvalue. It is not allowed by the standard to not move a moved argument, or forward a forwarded argument.

Reply:
What do you mean not allowed by standard? I do not think standard forbid you to capture a `rvalue` but not to forward or move them. It is just a (possibly false) warning from `clang-tidy` right?